lib/utils/docker.coffee

```diff
@@ -110,13 +110,33 @@ exports.tarDirectory = tarDirectory = (dir) ->
 		pack.finalize()
 		return pack
 
+cacheHighlightStream = ->
+	colors = require('colors')
```

Comment:
Can you use `colors/safe` to avoid messing with the string prototype (you already use the "safe" api below)